src/main/java/com/laserfiche/repository/api/clients/impl/model/APIServerException.java

```diff
@@ -2,26 +2,175 @@
 
 @javax.annotation.Generated(value = "io.swagger.codegen.v3.generators.java.JavaClientCodegen", date = "2022-08-19T17:07:43.799-04:00[America/New_York]")
 @JsonInclude(JsonInclude.Include.NON_NULL)
 public class APIServerException {
 
     @JsonProperty("operationId")
-    public String operationId = null;
+    private String operationId = null;
 
     @JsonProperty("message")
-    public String message = null;
+    private String message = null;
 
     @JsonProperty("errorCode")
-    public Integer errorCode = null;
+    private Integer errorCode = null;
 
     @JsonProperty("errorClass")
-    public String errorClass = null;
+    private String errorClass = null;
 
     @JsonProperty("statusCode")
-    public Integer statusCode = null;
+    private Integer statusCode = null;
 
     @JsonProperty("errorSource")
-    public String errorSource = null;
+    private String errorSource = null;
+
+    public APIServerException operationId(String operationId) {
+        this.operationId = operationId;
+        return this;
+    }
+
+    @Schema(description = "")
+    public String getOperationId() {
+        return operationId;
+    }
+
+    public void setOperationId(String operationId) {
+        this.operationId = operationId;
+    }
+
+    public APIServerException message(String message) {
+        this.message = message;
+        return this;
+    }
+
+    @Schema(description = "")
+    public String getMessage() {
+        return message;
+    }
+
+    public void setMessage(String message) {
+        this.message = message;
+    }
+
+    public APIServerException errorCode(Integer errorCode) {
+        this.errorCode = errorCode;
+        return this;
+    }
+
+    @Schema(description = "")
+    public Integer getErrorCode() {
+        return errorCode;
+    }
+
+    public void setErrorCode(Integer errorCode) {
+        this.errorCode = errorCode;
+    }
+
+    public APIServerException errorClass(String errorClass) {
+        this.errorClass = errorClass;
+        return this;
+    }
+
+    @Schema(description = "")
+    public String getErrorClass() {
+        return errorClass;
+    }
+
+    public void setErrorClass(String errorClass) {
+        this.errorClass = errorClass;
+    }
+
+    public APIServerException statusCode(Integer statusCode) {
+        this.statusCode = statusCode;
+        return this;
+    }
+
+    @Schema(description = "")
+    public Integer getStatusCode() {
+        return statusCode;
+    }
+
+    public void setStatusCode(Integer statusCode) {
+        this.statusCode = statusCode;
+    }
+
+    public APIServerException errorSource(String errorSource) {
+        this.errorSource = errorSource;
+        return this;
+    }
+
+    @Schema(description = "")
+    public String getErrorSource() {
+        return errorSource;
+    }
+
+    public void setErrorSource(String errorSource) {
+        this.errorSource = errorSource;
+    }
+
+    @Override
+    public boolean equals(java.lang.Object o) {
```

Comment:
maybe it's better to use only "Object" instead of the fully qualified name "java.lang.Object".

Reply:
Hi Samad, this code is from Swagger codegen and they are template-based code generators unlike us. I think they don't have a safe way of resolving import statements so they did the lazy way: fully qualified name.

Reply:
I see. yeah. Thanks for the feedback.